Portal/src/Datahub.Core/Components/AuthViews/DatahubAuthView.razor

```diff
@@ -81,16 +84,16 @@
             case AuthLevels.Personal:
                 return $"{UserGraphId},{RoleConstants.DATAHUB_ROLE_ADMIN}";
```

Comment:
I think for line 85, you can use 'datahubAdminRole' instead of 'RoleConstants.DATAHUB_ROLE_ADMIN'.
You may want create a variable for line 95 with RoleConstants.DATAHUB_ROLD_ADMIN_AS_GUEST .. incase that role gets used again in the future.

Reply:
Sounds good! I've cleaned up that section and made them all consistent.